File: 프로그래머스/lv1/12910. 나누어 떨어지는 숫자 배열/나누어 떨어지는 숫자 배열.kt
class Solution {
    fun solution(arr: IntArray, divisor: Int): IntArray {
        val list = mutableListOf<Int>()

        repeat(arr.size) {
            if(arr[it] % divisor == 0) list.add(arr[it])
        }
        if (list.size == 0) list.add(-1)
        return list.toIntArray().sortedArray()
    }
}

File: 프로그래머스/1/12910. 나누어 떨어지는 숫자 배열/나누어 떨어지는 숫자 배열.swift
func solution(_ arr:[Int], _ divisor:Int) -> [Int] {
    var answer: [Int] = []
    
    for i in arr {
        if i % divisor == 0 {
            answer.append(i)
        }
    }
    
    if answer.isEmpty { return [-1] }
    return answer.sorted()
}